### PR TITLE
ASAP-6 [Array] Find the number of unique data in array

### PR DESCRIPTION
**Pull request by ap-enign**
In this PR, I have suggested a solution to find the number of unique data in an array.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -9,3 +9,7 @@ add_subdirectory(Exercise_1)
 add_subdirectory(Exercise_2)
 add_subdirectory(Exercise_3)
 # add_subdirectory(Exercise_4)
+
+
+
+# Path to github of the project: https://github.com/ap-enign/Exercises
```
Comment (abrsaber) [inline CMakeLists.txt]: Move to readme file

File: Exercise_3/main.cpp [hunk 2]
```diff
@@ -1,49 +1,110 @@
...
-    for(int elm =0; elm <cnt; elm++){
+    for (int elm = 0; elm < cnt; elm++)
+    {
         std::cout << uni_arr[elm] << " ";
     }
     // ******************************
 
+    // *************************  AHMAD *************************
+    // The array that is going to store unique data
+    int unique_data[collection_size];
```
Comment (abrsaber) [inline Exercise_3/main.cpp:55]:
```suggestion
    int unique_arr[collection_size];
```

File: CMakeLists.txt (hunk 1 above)
Comment (ap-enign) [reply CMakeLists.txt]: Done

File: Exercise_3/main.cpp (hunk 2 above)
Comment (abrsaber) [reply Exercise_3/main.cpp:55]: The name should be understandable.

**Review by abrsaber — CHANGES_REQUESTED** [review]

File: Exercise_3/main.cpp (hunk 2 above)
Comment (ap-enign) [reply Exercise_3/main.cpp:55]: We can not define same name for different array name.

Comment (abrsaber) [reply Exercise_3/main.cpp:55]: you can use unique_array or something like that.

File: CMakeLists.txt (hunk 1 above)
Comment (abrsaber) [inline CMakeLists.txt]:
```suggestion

```

File: Exercise_3/main.cpp (hunk 2 above)
Comment (ap-enign) [reply Exercise_3/main.cpp:55]: My suggestion is to use unique_data as it is.

**Review by abrsaber — APPROVED** [review]

**Review by abrsaber — CHANGES_REQUESTED** [review]
Chack the last comment as well.

**Pull request merged**